src/com/zoerAleksandr/lesson5/Main.java

```diff
@@ -0,0 +1,97 @@
+package com.zoerAleksandr.lesson5;
+
+/**
+ * Project GeekBrainsLesson
+ *
+ * @Author Zoer Aleksandr
+ * Created 17.03.2021 15:45
+ */
+class Main {
+    public static void main(String[] args) {
+        Animal[] animals = {
+                new Dogs("Барбос", 10, 2),
+                new Dogs("Пегас", 9, 3),
+                new Cats("Пушок", 2, 0),
+                new Cats("Васька", 3, 0),
+        };
+        System.out.printf("Всего животных - %d \nИз них собак - %d, кошек - %d.\n\n", Animal.getCountAnimal(),
```

Comment:
I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [FS: Format string should use %n rather than n (VA_FORMAT_STRING_USES_NEWLINE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#fs-format-string-should-use-n-rather-than-n-va-format-string-uses-newline).
This format string includes a newline character (\n). In format strings, it is generally  preferable to use %n, which will produce the platform-specific line separator.